lib/wanda/executions/server.ex

```diff
@@ -32,7 +32,7 @@ defmodule Wanda.Executions.Server do
     checks =
       targets
       |> Target.get_checks_from_targets()
-      |> Catalog.get_checks()
+      |> Catalog.get_checks(env)
```

Comment:
As a comment, with this we are not simply filtering the catalog checks, we are filtering the execution as well, and I'm not sure if this was part of the initial idea.

Reply:
it wasn't, but since it came free without writing code I ask @dottorblaster  to do the filtering here.
wdyt?

Reply:
It is fine, but I think this should be documented somewhere, as It can lead to confusion

Reply:
I think we can document that the general behavior is  "best effort". We do the same for malformed or non-existing checks. @dottorblaster proposes to add a field in the result with the skipped checks this is an interesting option as well (it could also help us remove non-existing checks from the check selection, the migration we were talking about).

Reply:
Ok, let's keep in the radar. In the current runner we have the skipped thing, but It never worked out pretty well.
Let's go with the current approach. I just only wanted to highlight that these things must be of general knowledge

Reply:
@dottorblaster could we document the best effort approach in the Execution Server module documentation?